### ISSUE TITLE
Tune workspace dashboard to enable comparison across models and experiments

### ISSUE THREAD

**Issue by eu9ene** [mozilla/firefox-translations-training#408]
<img width="2545" alt="Screenshot 2024-01-29 at 9 43 07 AM" src="https://github.com/mozilla/firefox-translations-training/assets/2486505/549e3b27-eb8f-4446-b26e-220a8cff234d">

We have a great bar chart to compare across the models for one experiment (runs group).  We should figure out how to tune this dashboard or rename the steps so that we can also compare metrics as well as charts across the experiments. The expectation was that this tool will replace Tensorboard which does a good job visualizing lines from different runs on one dashboard.

<img width="2297" alt="Screenshot 2024-01-29 at 11 19 11 AM" src="https://github.com/mozilla/firefox-translations-training/assets/2486505/d445fada-e1e4-446e-979c-2cd3761ba824">

**Comment by eu9ene** [comment]
If W&B doesn't allow this, the workaround would be to rename runs with a suffix with experiment name so that they are unique. For example `student-finetuned_opustrainer`.

**Comment by eu9ene** [comment]
We discussed that W&B uses display names to show charts and since those names are not unique it doesn't show the runs properly on the charts. We discussed the following potential solutions:
1. We should figure out if there is a workaround on the W&B side, like using an internal ID that also includes a group name. 
2. Add a short suffix to each run name composed of a hash of the group name. We don't have a lot of groups per language, so the chance of a collision is pretty low and we can shorten the suffix. For example: `teacher-1_s7g3j`.
3. Just use a counter for the groups per project and add a suffix with the current value. For example `teacher-1_13`. We would need to load the project groups to calculate the value each time we need to publish something. There might be race conditions possible in this case when we run multiple tasks at the same time, for example, if we run many evaluations all at once as the only tasks in the group. So the hashing approach seems to be more robust.
